_config.yml

```diff
@@ -34,6 +34,7 @@ external_urls:
   previous_pytorch_versions: https://pytorch.org/previous-versions/
   udacity_courses: https://pytorch.org
   youtube: https://www.youtube.com/pytorch
+  forums: https://discuss.pytorch.org/
```

Comment:
It looks like forums link is already present as `discuss` on line 32. Can you use that instead? Then this PR is good to go.